src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/client/payhub/PayHubClient.java

```diff
@@ -27,4 +29,17 @@
         @RequestHeader("ServiceAuthorization") String serviceAuthorisation,
         @RequestBody PaymentRequest paymentRequest
     );
+
+    @RequestMapping(
+        method = RequestMethod.PUT,
+        value = "/bulk-scan-payments",
+        consumes = MimeTypeUtils.APPLICATION_JSON_VALUE,
+        produces = MimeTypeUtils.APPLICATION_JSON_VALUE
+    )
+    ResponseEntity<Void> putPayments(
```

Comment:
Method name should be 'updatePayments'. Let's use business related names rather than tech related.

Reply:
changed